SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -18,6 +18,10 @@
 #include "GameFramework/PlayerController.h"
 #include "UObject/UObjectIterator.h"
 
+#if WITH_UNREAL_DEVELOPER_TOOLS || (!UE_BUILD_SHIPPING && !UE_BUILD_TEST)
```

Comment:
Should this be WITH_GAMEPLAY_DEBUGGER?

Reply:
I copied this from `SpatialActorUtils.h`, and I guess I'll change it there as well!